Deeploy/Targets/Generic/Layers.py

```
@@ -709,3 +712,99 @@ def computeOps(self):
+
+
+class ExpLayer(ONNXLayer):
+
+    def computeOps(self):
+        return self.mapper.parser.operatorRepresentation['size']
+
+
+class SigmoidLayer(ONNXLayer):
+
+    def computeOps(self):
+        # σ(x) = 1 / (1 + exp(-x)): neg, exp, add, div
```

Ruff (0.15.12)

[warning] Comment contains ambiguous `σ` (GREEK SMALL LETTER SIGMA). Did you mean `o` (LATIN SMALL LETTER O)?

(RUF003)